src/_posts/databases/mysql/2000-01-01-connecting.md

```diff
@@ -0,0 +1,137 @@
+would have to update your code with the new value and trigger a new deployment,
+which would most probably contribute to a greater downtime.
+
+In most cases, you can pass the environment variable directly to the client
+library you are using. But sometimes, the library requires a specific URI
+format, invididual keypairs or another format. In such cases, you
+need to parse the connection URI to retrieve the required values and build
+what's required by the library you are using. Our advice to use the environment
+variable still remains applicable.
+
+
+## Getting the Connection URI
+
+### Using the Dashboard
+
+1. From your web browser, open your [dashboard](https://dashboard.scalingo.com)
+2. Click on the application for which you want to get the connection URI
+3. Click on the **Environment** tab
+4. Locate the `SCALINGO_MYSQL_URL` line and place your mouse over it to unblur
+   the value
+
+### Using the Command Line
+
+1. Make sure you have correctly [setup the Scalingo command line tool]({% post_url platform/cli/2000-01-01-start %})
+2. From the command line, get the environment variable value:
+   ```bash
+   scalingo --app my-app env-get SCALINGO_MYSQL_URL
+   ```
+   The output is:
+   ```bash
+   mysql://my_app_wxyz:Q7X7CU2vGjFiLZrA43OG@7e10d74b-c766-40b3-8dad-ce9cfa461311.my-app-wxyz.mysql.a.osc-fr1.scalingo-dbs.com:31000/my_app_wxyz?useSSL=true&verifyServerCertificate=false
+   ```
+
+
+## Enforcing TLS Connection
+
+All MySQL® addons [support TLS](https://dev.mysql.com/doc/refman/8.0/en/encrypted-connections.html)
+to encrypt all the network traffic between the client and the server.
+
+By default, all new MySQL® databases have TLS **available**,
+which means that you don't have to do anything to connect via TLS with a tool
+or library that uses TLS by default.
+
+If you want to enforce TLS connections to your database (so that TLS is
+not only available, but **mandatory**):
+
+1. From your web browser, [open your database dashboard]({% post_url databases/mysql/2000-01-01-getting-started %}#accessing-the-scalingo-for-mysql-dashboard)
+2. Select the **Overview** tab
+3. Locate the **TLS/Internet Access** block
+4. Toggle the corresponding option:
+
+{% assign img_url = "https://cdn.scalingo.com/documentation/screenshot_database_mongo_force_tls.png" %}
+{% include mdl_img.html %}
+
+
+Once this option is activated, the database denies any non-TLS connection.
+Consequently, your application must be configured to use TLS when connecting to
+the database.
+
+Some existing databases may not have TLS support activated yet. To activate
+TLS, you need to restart the database. Any action leading to a database
+restart (such as a plan update or an upgrade of the database) will activate
+TLS.
+
+Please note that while we strongly advise to use TLS, it remains an option,
+meaning that you can still access your database without it if needed.
+
+
+## Connecting Multiple Applications to the Same MySQL® Addon
```

Comment:
suggestion: add again the warning about the fact that if the connection string changes, it will require manual intervention to propagate this change?